### ISSUE TITLE
Animations of type Shape keys in Blender, do not play when loaded in Threejs using gLTF loader.

### ISSUE THREAD

**Issue by tauroamp** [mrdoob/three.js#12152]
Hi!
I'm having trouble playing an animation based on Blender's Shape Keys.
I've seen examples of @mrdoob and they all work without problems.
But something must be doing wrong in my case, the animations do not run on Threejs.
In this case, I am using the gLTF format with its corresponding loader in Threejs.
My question is if there is any problem with the gLTF loader or is it that the Shape Keys have to be built in a specific way in Blender?

Here is the thread where this problem is discused right now.
https://github.com/KhronosGroup/glTF-Blender-Exporter/issues/36#issuecomment-328074293

Thanks in advance!

**Comment by tauroamp** [comment]
Hi!
After a day reviewing every detail, I have discovered that Suzanne's animation is not shown because the corresponding "Shape Key" has the "Bezier" value in the type of interpolation.
When I change its value to "Linear", then the animation is reproduced.
I have tested it with other values (not all) and the animation is also achieved.
I really do not know the processing done by the GLTF exporter or its corresponding loader in Threejs, but I think this detail could help optimize them.
Thank you all.
